mlflow/tracking/_tracking_service/client.py

```diff
@@ -278,21 +281,44 @@ def log_metric(self, run_id, key, value, timestamp=None, step=None):
                       may support larger values.
         :param timestamp: Time when this metric was calculated. Defaults to the current system time.
         :param step: Training step (iteration) at which was the metric calculated. Defaults to 0.
+        :param synchronous: [Experimental] Defaults to True and no behavior change.
```

Comment:
Nit: "Behavior change" is a bit unclear (what version(s) does it refer to)? I think we can just remove that from the docstring.

```suggestion
        :param synchronous: [Experimental] Defaults to ``True``
```

Reply:
Can we make this change to other method docstrings?